src/services/send-audit-events.ts

```diff
@@ -63,15 +77,53 @@ export async function sendAuditEvent(
  * Helper function to build extension object based on the type of event
  * @param event - Original event received from TxMA
  * @param eventEnum - Event name as an EventEnum
- * @param appliedAt - optional timestamp of when then event transition was applied
+ * @param finalState - Final state after intervention was/ was not applied
+ * @param txmaEventName - The name of the TxMA event name
  * @returns - TxMAEgressExtensions object
  */
-function buildExtensions(event: TxMAIngressEvent, eventEnum: EventsEnum, appliedAt?: number): TxMAEgressExtensions {
+function buildExtensions(
+  event: TxMAIngressEvent,
+  eventEnum: EventsEnum,
+  finalState: AccountStateEngineOutput,
+  txmaEventName: TxMAEgressEventName,
+): TxMAEgressExtensions {
   return {
-    eventType: userLedActionList.includes(eventEnum) ? 'USER_LED_ACTION' : 'TICF_ACCOUNT_INTERVENTION',
-    event: eventEnum,
+    trigger_event: event.event_name,
+    trigger_event_id: event.event_id ?? 'UNKNOWN',
+    description: userLedActionList.includes(eventEnum) ? 'USER_LED_ACTION' : finalState.interventionName!,
     intervention_code: event.extensions?.intervention?.intervention_code,
     reason: event.extensions?.intervention?.intervention_reason,
-    appliedAt,
+    allowable_interventions: finalState.nextAllowableInterventions.filter(
+      (intervention) => !nonInterventionsCodes.has(intervention),
+    ),
+    state: buildStateAttribute(finalState, eventEnum, txmaEventName),
+    action: buildActionAttribute(finalState, eventEnum),
   };
 }
+
+function buildStateAttribute(
+  finalState: AccountStateEngineOutput,
+  eventEnum: EventsEnum,
+  txmaEventName: TxMAEgressEventName,
+): State | undefined {
+  if (!finalState.finalState.suspended || (finalState.finalState.suspended && userLedActionList.includes(eventEnum)))
```

Comment:
should this be `!` ? I am thinking the first part of this statement would evaluate to true if the blocked: true and suspended: false... and return a ACTIVE status. Apols if reading it wrong